### PR TITLE
Feature/forward camera info

### PR DESCRIPTION
**Pull request by floriantschopp**

### CODE REVIEW

File: versavis/include/versavis/versavis_synchronizer.h [hunk 1]
```diff
@@ -56,40 +58,50 @@ class VersaVISSynchronizer {
...
+  sensor_msgs::CameraInfo camera_info_msg_;
   ros::Time last_stamp_;
   ros::Time init_timestamp_;
 
   // Constants.
   const uint8_t kMaxImageCandidateLength;
   const uint8_t kMinSuccessfullConsecutiveMatches;
-  const double kMaxImageDelayThreshold;
-  const uint8_t kSlowPublisherImageCounterThreshold;
+  const double kMaxImageDelayThreshold;;
```
Comment (floriantschopp) [inline versavis/include/versavis/versavis_synchronizer.h]:
```suggestion
  const double kMaxImageDelayThreshold;
```

File: versavis/include/versavis/versavis_synchronizer.h [hunk 2]
```diff
@@ -56,40 +58,50 @@ class VersaVISSynchronizer {
...
 
   // Image numbers and initialization.
   uint8_t init_number_;
   uint64_t last_image_number_;
   int64_t offset_;
   bool initialized_;
 
-  // Conifugration.
+  // Configuration.
```
Comment (floriantschopp) [inline versavis/include/versavis/versavis_synchronizer.h:100]: +1

File: versavis/src/versavis_synchronizer.cpp [hunk 3]
```diff
@@ -39,14 +52,28 @@ VersaVISSynchronizer::VersaVISSynchronizer(const ros::NodeHandle &nh,
   // Publisher to let the triggering board know as soon as the camera is
   // initialized.
   initialized_pub_ = nh_.advertise<std_msgs::Bool>(initialized_pub_topic_, 1u);
-  ROS_INFO("Publishing to %s.", initialized_pub_topic_.c_str());
+  ROS_INFO("Publishing initialization status to %s.",
+           initialized_pub_topic_.c_str());
 
   image_fast_pub_ = image_transport_.advertise(image_fast_pub_topic_, 10u);
-  ROS_INFO("Publishing to %s.", image_fast_pub_topic_.c_str());
+  ROS_INFO("Publishing image o %s.", image_fast_pub_topic_.c_str());
```
Comment (floriantschopp) [inline versavis/src/versavis_synchronizer.cpp]:
```suggestion
  ROS_INFO("Publishing image to %s.", image_fast_pub_topic_.c_str());
```

**Review by BenzStefanBosch — APPROVED** [review]
Thanks for the heads-up. Looks exactly like what I did, but also takes slow publisher into account.

**Pull request merged**